Gems/OpenXRVk/Code/Source/Platform/Common/Default/OculusTouch_Default.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <OpenXRVk/InputDeviceXRController.h>
+#include <AzCore/std/containers/unordered_map.h>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
[this is just my opinion, feel free to ignore] IMO adding this long `//////` lines just bloats the file unnecessarily and makes the code harder to maintain, it's not clear when to add, when not to add, what length should it have...

Reply:
Yeah, maybe I use it too much.  There's lots of code in O3DE that makes use of this [kind of divider](https://github.com/o3de/sig-core/blob/main/governance/API-Ref-Guidelines-Update.md#section-comments), and I've continued using it myself.